lighthouse-core/test/gather/driver-test.js

```diff
@@ -587,6 +604,7 @@ describe('.gotoURL', () => {
       driver._waitForNetworkIdle = createMockWaitForFn();
       driver._waitForCPUIdle = createMockWaitForFn();
 
+      // @ts-ignore
```

Comment:
is this so we don't need to create an entire passcontext?

Reply:
ya